pages/apis/rest_api/analytics/suites.md

```diff
@@ -98,7 +104,9 @@ Optional [request body properties](/docs/api#request-body-properties):
         <em>Example:</em> <code>"team_ids": ["3f4aa5ee-671b-41b0-9b44-b94831db6cc8"]</code></td></tr>
       </td>
     </tr>
-    <tr>
+    <tr><th><code>application_name</code></th><td>Application name for the suite.<br><em>Example:</em> <code>"Buildkite"</code></td></tr>
+    <tr><th><code>color</code></th><td>Color for the suite navatar.<br><em>Example:</em> <code>"#FFF700"</code></td></tr>
+    <tr><th><code>emoji</code></th><td>Emoji for the suite navatar.<br><em>Example:</em> <code>"🍋"</code>, <code>"\:lemon\:"</code></td></tr>
```

Comment:
minor nit: do we need to [link to which emojis are supported](https://github.com/buildkite/emojis#emoji-reference)?

Reply:
Ooh didn't know we had this, will add!